package.json

```diff
@@ -1,26 +1,52 @@
 {
   "name": "webpack-long-term-cache-demo",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
Ignore this. I just wanted to make clear that those changes is breaking everything 😸